File: NetCoreGameServer/NetCoreGameServer/Data/Network/RequestPacketType.cs
﻿namespace NetCoreGameServer.Data.Network;

public enum RequestPacketType
{
    Ping = 0,
    Pong = 1,
    ListCharacters = 2,
    CreateCharacter = 3,
    SendChatMessage = 4,
    SelectCharacter = 5,
    CreateGroup = 6,
    ListGroups = 7,
    JoinGroup = 8,
    LeaveGroup = 9,
    GetUser = 10,
    SetGroupLeader = 11,
    UpdateCharacter = 12,
    EquipItem = 13,
    UnequipItem = 14,
    DeleteItem = 15,
    UpdateMaze = 16,
    MoveDirection = 17,
    AttackTarget = 18,
}

File: next/src/network/RequestPacketType.ts
enum RequestPacketType {
  Ping,
  Pong,
  ListCharacters,
  CreateCharacter,
  SendChatMessage,
  SelectCharacter,
  CreateGroup,
  ListGroups,
  JoinGroup,
  LeaveGroup,
  GetUser,
  SetGroupLeader,
  UpdateCharacter,
  EquipItem,
  UnequipItem,
  DeleteItem,
  UpdateMaze,
  MoveDirection,
  AttackTarget,
  ChangeZone
}

export default RequestPacketType;